qa/L0_backend_python/python_test.py

```diff
@@ -265,7 +265,7 @@ def test_unicode(self):
         with httpclient.InferenceServerClient("localhost:8000") as client:
             utf8 = '😀'
             input_data = np.array([bytes(utf8, encoding='utf-8')],
-                                  dtype=np.bytes_)
+                                  dtype=np.object_)
```

Comment:
Does bytes_ not work anymore? If so that would break backwards compat. If both work then should have a test for both.

Reply:
It works, if it doesn't end with `\x00`. I will add tests in `L0_string_io` to test for both cases.